pages/seller/order/detail/[id].js

```diff
@@ -0,0 +1,28 @@
+import SellerLayout from '@components/seller/SellerLayout';
+import SiteHead from '@components/common/SiteHead.js';
+import SellerOrderList from '@components/order/SellerOrderList';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+
+export default function Home() {
+  const router = useRouter();
+  const [sellerId, setSellerId] = useState(router.query.id);
+
+  useEffect(() => {
+    setSellerId(router.query.id);
+  }, [router]);
+
```

Comment:
다음과 같은 warning을 내뿜더라구요 아래코드처럼 바꿔주면 warning 없어집니다.
<img width="1479" alt="image" src="https://user-images.githubusercontent.com/52902010/194698184-04383dfc-a6df-428f-acf5-abf5a3ceb138.png">


```suggestion
  useEffect(() => {
    setSellerId(router.query.id);
  }, [router.query]);

```

설명하자면

router가 큰 객체라서 경고를 내뿜는데요, query로 범위를 축소하는 거에요!